shared/fs/index.js

```diff
@@ -1,30 +1,110 @@
 // @flow
-import * as React from 'react'
-import {globalStyles, globalMargins} from '../styles'
-import {Box, Button, Text, ButtonBar} from '../common-adapters'
-
-type Props = {
-  counter: number,
-  you: ?string,
-  increase: () => void,
-  increase10: () => void,
+import React, {PureComponent} from 'react'
+import {FolderVisibility} from '../constants/types/fs'
+import {globalStyles, globalColors, globalMargins} from '../styles'
+import {isMobile} from '../constants/platform'
+import {Box, Button, ClickableBox, Icon, List, Text} from '../common-adapters'
+import {type IconType} from '../common-adapters/icon'
+
+const stylesCommonCore = {
+  alignItems: 'center',
+  borderBottomColor: globalColors.black_05,
+  borderBottomWidth: 1,
+  justifyContent: 'center',
 }
 
-const Fs = ({counter, you, increase, increase10}: Props) => (
-  <Box style={containerStyle}>
-    <Text type="Header">Hi {you}!</Text>
-    <Text type="BodySemibold">Count: {counter}</Text>
-    <ButtonBar align="flex-start">
-      <Button type="Primary" onClick={increase} label="Up by 1" />
-      <Button type="Primary" onClick={increase10} label="Up by 10" />
-    </ButtonBar>
-  </Box>
-)
+const stylesCommonRow = {
+  ...globalStyles.flexBoxRow,
+  ...stylesCommonCore,
+  minHeight: isMobile ? 64 : 48,
+  padding: 8,
+}
 
-const containerStyle = {
+const stylesContainer = {
   ...globalStyles.flexBoxColumn,
+  ...globalStyles.fullHeight,
   flex: 1,
-  padding: globalMargins.xlarge,
 }
 
+const rootFolders = [
```

Comment:
So this is great for the skeleton and keeping things simple. Just a heads up that I would assume the actual implementation would have connected rows and the props that would be passed to the children would be mostly ids (or paths or whatever the identifier is)

data like rootFolders would typically be derived from the store and created by the connect wrapper